assets/src/edit-story/elements/text/edit.js

```diff
@@ -250,6 +251,13 @@ function TextEdit({
     wrapper.style.height = `${editorHeightRef.current}px`;
   }, [editorState]);
 
+  // Update the height when the text height updated from the outside like design panel
+  useEffect(() => {
```

Comment:
Why not just add `eHeight` dependency to the previous `useEffect` which is already changing the height?

Reply:
Correct. textEditorRef will be updated. Thank you!